src/sparseml/export/export.py

```diff
@@ -382,20 +387,20 @@ def export(
     default=None,
     help=(
         "Integration the model was trained under. "
-        "ie transformers, image-classification. Will be inferred by default"
+        "i.e. transformers, image-classification. Will be inferred by default"
     ),
 )
 @click.option(
     "--sample_data",
     type=str,
     default=None,
-    help="Path to sample data to export with. default None",
+    help="Path to sample data to export with. Default - None",
 )
 @click.option(
     "--task",
     type=str,
     default=None,
-    help="subtask within the integration this model was trained on. Default - None",
+    help="Task within the integration this model was trained on. Default - None",
```

Comment:
Better idea: to avoid importing to this general module directly from transformers, let's print tasks are available once they call the export. This way we can keep all the transformers-related items within the transformer subrepo. See my recent commit.